docs/development.rst

```diff
@@ -37,12 +37,68 @@ To run a single test, use this following command, replace the test case name at
     make test TESTS=tests/test_api.py::TestAPI::test_error_unencrypted_reply
 
 
-To test against a live development server, you will need to run the SecureDrop developent
-container from the main SecureDrop repository on your host. This can be done via ``make -C securedrop dev``.
+To test against a live development server, you will need to run the SecureDrop
+developent container from the main SecureDrop repository on your host. This
+can be done via ``NUM_SOURCES=5 make -C securedrop dev``.
+
+In this repo, comment out the ``@vcr`` decorator of the ``setUp`` method in
+`test_api.py` and execute which ever tests you want to run. If you want to
+re-run all tests against the API, remove all the ``.yml`` files in the
+``data`` directory. 
+
+
+Gererating test data for `APIProxy`
+-----------------------------------
+
+
+To test or to generate new test data file for the `APIProxy` class in
+`test_apiproxy.py` file, you will have to setup
+[QubesOS](https://qubes-os.org) system.
```

Comment:
This is markdown syntax for links, not RST.

Reply:
Will fix it, thanks for pointing out.

Reply:
I think you forgot to push your changes because this is still markdown.